### PR TITLE
Docs / update docs for workspaces

### PR DESCRIPTION
**Pull request by ChristiaanScheermeijer**
## Description

In the last sprint, I already changed most of the documentation. During this sprint, @langemike reviewed this and made some changes.

> Note: this branch is rebased onto the `feat/accessibility-and-bug-fixes` branch

### CODE REVIEW

**Comment by ChristiaanScheermeijer** [comment]
@AntonLantukh, PR #456 was deleted due to the merge of #451. We will address your feedback in this PR.

**Comment by ChristiaanScheermeijer** [comment]
@AntonLantukh I've looked at the feedback and fixed the issues. I added the package `web` shortcut and a small section in the readme.

I also did a quick spell check and noted that my editor really wanted to apply structural wrapping to make it easier to read. We could apply this to all files at once, but this will change basically every documentation in the repo 😅 

image

File: docs/translations.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Before changing or adding languages, read the sections below to understand the m
 
 ## Translation status
 
-Here is a list of all supported translations included in the OTT Web App. The status indicates if the translations are 
+Here is a list of all supported translations included in the OTT Web App. The status indicates if the translations are
```
Comment (AntonLantukh) [inline docs/translations.md:9]: @ChristiaanScheermeijer do you think we should move this file to the `web` folder? Or do you have plans to move translations out of the web to reuse in other implementations?

Comment (AntonLantukh) [reply docs/translations.md:9]: Here is the answer probably:

```
> The translation files will be moved to the `./packages/i18n` package in the near future.
```

**Review by AntonLantukh — APPROVED** [review]
Thank you!

**Pull request merged**